cmd/broker/update_test.go

```diff
@@ -2393,7 +2395,8 @@ func TestUpdateBTPOperatorCredsSuccess(t *testing.T) {
 	i, err = suite.db.Instances().GetByID(id)
 	assert.NoError(t, err, "getting instance after update")
 	assert.ElementsMatch(t, rsu2.KymaConfig.Components, []*gqlschema.ComponentConfigurationInput{})
-	assert.ElementsMatch(t, componentNames(rsu2.ClusterSetup.KymaConfig.Components), []string{"ory", "monitoring", "btp-operator"})
+	assert.ElementsMatch(t, componentNames(rsu2.ClusterSetup.KymaConfig.Components),
+		[]string{"cluster-essentials", "ory", "monitoring", "btp-operator"})
```

Comment:
The same question - do we still need ory?

Reply:
We don't need ory, but for tests it doesn't matter. See my other answer: https://github.com/kyma-project/kyma-environment-broker/pull/264#discussion_r1422517080